docs/sphinx/examples/cpp/algorithms/qaoa_maxcut.cpp

```diff
@@ -1,37 +1,37 @@
-// Compile and run with:
-// ```
-// nvq++ qaoa_maxcut.cpp -o qaoa.x && ./qaoa.x
-// ```
+/// Compile and run with:
+/// ```
+/// nvq++ qaoa_maxcut.cpp -o qaoa.x && ./qaoa.x
+/// ```
 
 #include <cudaq.h>
 #include <cudaq/algorithm.h>
 #include <cudaq/gradients.h>
 #include <cudaq/optimizers.h>
 #include <cudaq/spin_op.h>
 
-// Here we build up a CUDA Quantum kernel for QAOA with p layers, with each
-// layer containing the alternating set of unitaries corresponding to the
-// problem and the mixer Hamiltonians. The algorithm leverages the CUDA Quantum
-// VQE support to compute the Max-Cut of a rectangular graph illustrated below.
-//
-//        v0  0---------------------0 v1
-//            |                     |
-//            |                     |
-//            |                     |
-//            |                     |
-//        v3  0---------------------0 v2
-// The Max-Cut for this problem is 0101 or 1010.
+/// Here we build up a CUDA Quantum kernel for QAOA with p layers, with each
+/// layer containing the alternating set of unitaries corresponding to the
+/// problem and the mixer Hamiltonians. The algorithm leverages the CUDA Quantum
+/// VQE support to compute the Max-Cut of a rectangular graph illustrated below.
+///
+///        v0  0---------------------0 v1
+///            |                     |
+///            |                     |
+///            |                     |
+///            |                     |
+///        v3  0---------------------0 v2
```

Comment:
I'm a bit surprised this ASCII art picture passes a spelling check. But it's cool that it does.